<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineedit_position"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineedit_radius"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="shape_label">
      <property name="text">
       <string>Shape</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="color_label">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="radius_label">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_shape">
      <item>
       <property name="text">
        <string>circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>square</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="position_label">
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>position</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="button_color">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
